### PR TITLE
Med Pill Blue Pill

### LINKED ISSUE #4412: Mediborgs and pills
#### Brief description of the issue
1) mediborg feeds pill to patient successfully
2) mediborg then attempts to feed pill to patient again
3) if you don't move away in time, you "give them the pill again" and spawn an empty ghost pill
4) this empty ghost pill can't be grinded or fed to anyone, but it can be dissolved in a beaker to produce nothing
#### What you expected to happen
The mediborg feeds the pill to the patient once. The pill disappears. The patient consumes the pill.

#### What actually happened
The mediborg feeds the pill to the patient once. The pill disappears. The patient consumes the pill. The mediborg feeds the pill to the patient again. The pill reappears underneath the mediborg, but empty. The patient doesn't gain any additional reagents, but is probably confused.

#### Steps to reproduce
Be mediborg. Feed pill to patient. Feed pill to patient. Be confused.

#### Additional info:
 - Latest revision, not just on the pill branch.
 - Causes a runtime thing when you try and feed them an empty pill.

### PR DESCRIPTION
**Pull request by mistyLuminescence**
* Reworks some pill bottle code to be more consistent (absolute pathing and tabs instead of spaces).
* Adds Carbon pills (15u) and pill bottles; used to neutralise stomach reagents.
* Adds Iron pills (15u) and pill bottles; are used to aid in blood regeneration.
* Adds a new random spawn list - 'random pill bottle'. 20% each of Spaceacillin, Dermaline, Dexalin Plus, Bicaridine, Iron.
* Removes individual pill bottle entries from the medical random spawn list and replaces them with random pill bottles. Tramadol and Dylovene still have their own entries on the random spawn list to ensure greater chance of spawn, as does Carbon. Total chance of getting a pill bottle of some kind is a little lower.
* Dylovene and Inaprovaline syringes are marginally less common, and Spaceacillin syringes are marginally more common.
* Stasis bags, previously in the 'lite' medical spawn list, and body bags, previously in the regular medical spawn list, have been swapped around. Stasis bags aren't as common as body bags used to be, though, since they're actually pretty useful.
* The wall-mounted pill cabinets in Medical have had their contents removed from the map...
* ... and added to the code instead. A guaranteed spawn of Tramadol, Dylovene and Carbon pills, as well as a random pill bottle selected from the random list above.
* All roundstart medical pills now have the pill dosage in their examine text - e.g. "A broad-spectrum anti-toxin. Contains 15 units of Dylovene."
* Pill bottles can now be emptied onto the ground using the 'Empty' verb, available in the Object tab or by right-click.

What does this mean for medical?
* Basically, you get more pills (most notably Carbon and rarely Iron), so you don't have to break into Chemistry every round or risk everyone dying. Now it's just most rounds, because RNG hates you and will bombard you with cillin when you desperately need derma.
* You also get stasis bags instead of body bags for the heavy-duty med spawns (e.g. paramedic lockers), but you get body bags instead of stasis bags for the weaksauce med spawns (sorry, no more stasis bags lying around in maint!). Stasis bag spawns are accordingly a little rarer now.
* It's easier to see how much you're giving someone without having to memorise the pill size, since you can examine them now.
* Oh, and you can empty them onto the ground too. What can I say except, '[you're welcome](https://www.youtube.com/watch?v=79DijItQXMM)'?
* All this works for mediborgs too, and mediborgs no longer have that weird problem where they feed someone a pill twice and spawn a ghost pill beneath them ( fixes #4412 ).

also fight me travis.

### CODE REVIEW

**Comment by MisterLayne** [comment]
Looks good.

**Comment by Mechoid** [comment]
'Empty them onto the ground.' Wow it's like the mediborgs being able to hold them is going to have a purpose other than storing them.

**Comment by Belsima** [comment]
TheBluePill

**Comment by mistyLuminescence** [comment]
Dexalin? And mediborgs can't empty pill bottles, not sure they can access the verb at all - might be a way to change this but I'm not sure how

**Comment by mistyLuminescence** [comment]
Update: mediborgs can now pick up a pill bottle in their gripper, empty it by activating their gripper, and fill it by clicking pills on the ground; they can also pick up a pill and click someone with it to feed it to them (once, and without spawning any ghost pills - see #4412 ). Stasis bags and random pill bottles are now slightly rarer, and roller holder code has been cleaned up a bit.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -186,138 +187,166 @@
...
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+	new /obj/item/weapon/reagent_containers/pill/citalopram(src)
+
+/obj/item/weapon/storage/pill_bottle/carbon
+	name = "bottle of Carbon pills"
+	desc = "Contains pills used to neutralise chemicals in the stomach."
```
Comment (Belsima) [inline code/game/objects/items/weapons/storage/firstaid.dm]: Very minor and not required to be changed, `neutralize` seems to be the usual standard.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -98,7 +98,7 @@
 //Pills
 /obj/item/weapon/reagent_containers/pill/antitox
 	name = "Anti-toxins pill"
-	desc = "Neutralizes many common toxins."
+	desc = "Neutralizes many common toxins. Contains 25 units of Dylovene."
```
Comment (Belsima) [inline code/modules/reagents/reagent_containers/pill.dm]: How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -137,7 +137,7 @@
 
 /obj/item/weapon/reagent_containers/pill/stox
 	name = "Sleeping pill"
-	desc = "Commonly used to treat insomnia."
+	desc = "Commonly used to treat insomnia. Contains 15 units of Soporific."
```
Comment (Belsima) [inline code/modules/reagents/reagent_containers/pill.dm]: How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -147,7 +147,7 @@
 
 /obj/item/weapon/reagent_containers/pill/kelotane
 	name = "Kelotane pill"
-	desc = "Used to treat burns."
+	desc = "Used to treat burns. Contains 15 units of Kelotane."
```
Comment (Belsima) [inline code/modules/reagents/reagent_containers/pill.dm]: How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 5]
```diff
@@ -157,7 +157,7 @@
 
 /obj/item/weapon/reagent_containers/pill/paracetamol
 	name = "Paracetamol pill"
-	desc = "Tylenol! A painkiller for the ages. Chewables!"
+	desc = "Tylenol! A painkiller for the ages. Chewables! Contains 15 units of Paracetamol."
```
Comment (Belsima) [inline code/modules/reagents/reagent_containers/pill.dm]: How are you able to tell this just by examining it? I believe this was originally intentionally obfuscated. Also, TYLENOL is always capitalized.

File: code/modules/reagents/reagent_containers/pill.dm (hunk 2 above)
Comment (mistyLuminescence) [reply code/modules/reagents/reagent_containers/pill.dm]: Probably has a little '25' written on the back of the pill or something.

File: code/game/objects/items/weapons/storage/firstaid.dm (hunk 1 above)
Comment (mistyLuminescence) [reply code/game/objects/items/weapons/storage/firstaid.dm]: americentrism out

File: code/modules/reagents/reagent_containers/pill.dm (hunk 5 above)
Comment (mistyLuminescence) [reply code/modules/reagents/reagent_containers/pill.dm]: Just copied it from what it was before, the existence of Tylenol is completely coincidental and in no way related to TYLENOL®

Comment (Atermonera) [reply code/modules/reagents/reagent_containers/pill.dm]: Tylenol is just a brand name for the actual painkiller that's become synonymous with the painkiller itself in the US, like how band-aid is just a brand name for a bandage

Comment (Belsima) [reply code/modules/reagents/reagent_containers/pill.dm]: TYLENOL is just a brand name for paracetamol, isn't it? And we don't use 'band-aid' ingame so far as I can tell

Comment (Atermonera) [reply code/modules/reagents/reagent_containers/pill.dm]: Yeah, p sure it's paracetamol, and that was more an irl example

**Comment by mistyLuminescence** [comment]
PR's been up for a while now, any news on if this is something we like?

File: code/modules/reagents/reagent_containers/pill.dm (hunk 2 above)
Comment (Atermonera) [reply code/modules/reagents/reagent_containers/pill.dm]: Anewbe remains opposed to the labeled volumes, I have no preference.

File: code/modules/reagents/reagent_containers/pill.dm (hunk 5 above)
Comment (Atermonera) [reply code/modules/reagents/reagent_containers/pill.dm]: Just call it paracetamol, that's what it is everywhere else in the code.

**Comment by mistyLuminescence** [comment]
Tylenol doesn't exist any more, de Morgan gets a thumbs-up from the year 2561, and I'm hoping we can keep the pill amount labels in, since overdosing someone on pills by accident is really not fun. You can take two dexalin pills, two tramadol pills, two cillin pills, one and a half bicaridine pills, but only one dermaline pill, only one dexplus pill, for example.

**Pull request merged**